### DISCUSSION TITLE
OPA Support for JWT EdDSA

### DISCUSSION THREAD

**Discussion by DanishFayaz** [open-policy-agent/community#322] [OPA and Rego]
Does OPA have plans to add support for JWT EdDSA verification / encryption?

**Comment by srenatus** [comment]
Looking through the [documentation on JWT verification builtins](https://www.openpolicyagent.org/docs/latest/policy-reference/#tokens), the answer is currently **no**. Contributions welcome, of course 😃 Also, you could [open a Feature Request](https://github.com/open-policy-agent/opa/issues/new?assignees=&labels=feature-request&template=feature_request.md&title=) in the OPA repo.
